src/patternfly/components/DataList/examples/DataList.md

```diff
@@ -386,6 +386,183 @@ When a list item includes more than one block of content, it can be difficult fo
+              {{#> data-list-expandable-content-body data-list-expandable-content-body--modifier="pf-m-no-padding"}}
+                This expanded section has no padding.
+              {{/data-list-expandable-content-body}}
+            {{/data-list-expandable-content}}
+          {{/data-list-item}}
+        {{/data-list}}
+      {{/data-list-expandable-content-body}}
+    {{/data-list-expandable-content}}
+  {{/data-list-item}}
+
+  {{#> data-list-item data-list-item--id="item-2"}}
+    {{#> data-list-item-row}}
+      {{#> data-list-item-control}}
+        {{#> data-list-toggle button--attribute=(concat 'aria-labelledby="' data-list--id '-toggle2 ' data-list--id '-item2" id="' data-list--id '-toggle2" aria-label="Toggle details for" aria-expanded="false" aria-controls="' data-list--id '-content2"')}}{{/data-list-toggle}}
+      {{/data-list-item-control}}
+      {{#> data-list-item-content}}
+        {{#> data-list-cell}}
+          <div id="{{data-list--id}}-{{data-list-item--id}}">Secondary content</div><span>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</span>
+        {{/data-list-cell}}
+        {{#> data-list-cell}}
+          <span>Lorem ipsum dolor sit amet.</span>
+        {{/data-list-cell}}
+        {{#> data-list-cell}}
+          <span>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</span>
+        {{/data-list-cell}}
+      {{/data-list-item-content}}
+      {{#> data-list-item-action}}
+        {{#> data-list-action id=(concat data-list--id '-action2')}}{{/data-list-action}}
+      {{/data-list-item-action}}
+    {{/data-list-item-row}}
+    {{#> data-list-expandable-content data-list-expandable-content--attribute=(concat 'id="' data-list--id '-content2" aria-label="Secondary content details"')}}
+      {{#> data-list-expandable-content-body}}
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+      {{/data-list-expandable-content-body}}
+    {{/data-list-expandable-content}}
+  {{/data-list-item}}
+
+  {{#> data-list-item data-list-item--expanded="true" data-list-item--id="item-3"}}
+    {{#> data-list-item-row}}
+      {{#> data-list-item-control}}
+        {{#> data-list-toggle button--attribute=(concat 'aria-labelledby="' data-list--id '-toggle3 ' data-list--id '-item3" id="' data-list--id '-toggle3" aria-label="Toggle details for" aria-expanded="true" aria-controls="' data-list--id '-content3"')}}{{/data-list-toggle}}
+      {{/data-list-item-control}}
+      {{#> data-list-item-content}}
+        {{#> data-list-cell data-list-cell--modifier="pf-m-icon"}}
+          <i class="fas fa-code-branch" aria-hidden="true"></i>
+        {{/data-list-cell}}
+        {{#> data-list-cell}}
+          <div id="{{data-list--id}}-{{data-list-item--id}}">Tertiary content</div><span>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</span>
```

Comment:
Is this actually secondary? 
What do you think about some comments separating the various lists a bit, and some variation in the text so you can more easily tell what is what? It's quite difficult to sort through what's going on here to understand the structure.